back-end/src/main/kotlin/Gneiss/PacketCompiler/DatabaseAccess/UserDao.kt

```diff
@@ -31,30 +32,22 @@ class UserDao {
         val createAccountQuery = "INSERT INTO users (email, password, first_name, last_name, role_id) VALUES (?, ?, ?, ?, 'user')"
 
         val validateCredentialsQuery = "SELECT id, first_name, last_name, role_id FROM users WHERE email = ? AND password = ?"
+
+        val validateHashedCredentialsQuery = "SELECT id, password, first_name, last_name, role_id WHERE email = ?"
     }
 
     fun validateCredentials(email: String, password: String): CredentialsResponse {
         getConnection()
-        var prepStatement = connection!!.prepareStatement(validateCredentialsQuery)
+        var prepStatement = connection!!.prepareStatement(validateHashedCredentialsQuery)
         prepStatement.setString(1, email)
-        prepStatement.setString(2, password)
         var resultSet = prepStatement.executeQuery()
-
-        // Check how many rows are in the result set, if it equal to one it is a valid set of credentials
-        var rowCount = getRowCount(resultSet)
         connection!!.close()
```

Comment:
Did you manually run both these functions? There's an existing bug in validateCredentials that makes it non-functional.